packages/x-components/src/views/base-config.ts

```diff
@@ -6,6 +6,7 @@ import { realAdapter } from '../adapter/real-adapter';
 export const baseSnippetConfig: SnippetConfig = {
   instance: 'juguettos',
   lang: 'es',
+  env: 'staging',
```

Comment:
Is this right? Be careful

Reply:
We need to use staging in order to have Redirections in the layout view, also I don't like to use the customer production as default.